### ISSUE TITLE
Visual Accessibility option

### ISSUE THREAD

**Issue by subz390** [mdn/kuma#7334]
Please provide a Visual Accessibility (dark theme) option for your developer docs section.
https://developer.mozilla.org/en-US/docs/Web

**Summary**
_What problem would this solve?_
Improved Visual Accessibility to the valuable reference information contained at your resource.

**Audience**
_Who has this problem? Everyone? Article authors? Etc._
Everyone with Visual Accessibility needs, and everyone else who finds it easier on the eyes reading on a dark theme.

**Rationale**
_How do you know that people identified above actually have this problem?_
It's for people with Accessibility needs and written into the [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/0)

**Workaround**
_How are the people identified above currently handling this problem?_
I can't speak for how other people handle their Visual Accessibility needs
Workarounds aren't a reasonable option to accessibility needs.

**Proposal**
_Do you have suggestions for solving this problem?_
You employ a competent developer to create a dark theme with comfortable contrast.
Or better still, to create a customisable theme panel where one can define the standard colors and contrast options.

**Additional context**
_Is there anything else we should know?_
I love your developer documentation, it's the best there is on the internet, and I care about having comfortable access to it.

**Comment by schalkneethling** [comment]
Hey @subz390,

Do you think having the entire page in dark mode would be beneficial or would for example having all the code samples be in dark mode be helpful to you? Thank you

**Comment by subz390** [comment]
Hi @schalkneethling The whole page please - here's an example I recently created a theme for the [Svelte docs](https://svelte.dev/docs) site. 

![200706-100340 darkmode-screenshot](https://user-images.githubusercontent.com/11546815/87205114-8e1d7b80-c2fe-11ea-97c3-fba61544b4c5.png)

**Comment by atopal** [comment]
duplicate of https://github.com/mdn/kuma/issues/6880

**Comment by subz390** [comment]
Technically it's not a duplicate as I raise the point of `Accessibility Needs`, where as #6880 only raises the issue of `sensitive eyes`.  If you close this issue I feel you are ignoring this aspect of the argument are you not?
Whilst we're both requesting a dark mode, I feel you need to consider all the reasons.
If you are then that's fine, I just wanted to be clear or have it clarified that you are and whether you have already made a decision, thank you.

**Comment by atopal** [comment]
Hey Emil, the WCAG are fairly comprehensive, which part do you think we are violating or would better be addressed with a dark background?

**Comment by subz390** [comment]
Looking at the basics here, of typcial accessibility options.
Dark mode to reduce glare with a comfortable and increased contrast to improve character recognition.  For example a theme like this: https://devexpress.github.io/ThemeBuilder/master/material/blue-dark
And an option to adjust the font size.


https://www.w3.org/TR/WCAG21/#visual-presentation
https://www.w3.org/TR/WCAG21/#contrast-enhanced
https://www.w3.org/TR/WCAG21/#resize-text


Sorry I also see that my original link was broken.

**Comment by atopal** [comment]
Thanks for commenting further on this Emil. As much as possible we strive for AAA levels on WCAG, and we do try to keep a high contrast ratio for legibility, but I'm not convinced that dark mode would be required for or even benefit accessibility, and I haven't seen anything in the WCAG that would indicate otherwise.

**Comment by subz390** [comment]
Ok thanks for clarifying your position on the matter.
